lib/processor/xtensa/cpu.h

```diff
@@ -13,5 +13,6 @@
 #define __METAL_XTENSA_CPU__H__
 
 #define metal_cpu_yield()
+#define __sync_synchronize()
```

Comment:
any reason to not squash this commit with the previous one?

Reply:
Done.